xl_auth/templates/public/home.html

```diff
@@ -1,39 +1,64 @@
 <!-- public/home.html -->
 {% extends "layout.html" %}
 {% block content %}
-    <!-- Main jumbotron for a primary marketing message or call to action -->
-    <div class="jumbotron">
-
-        <h1>{{ _('Welcome to xl_auth') }}</h1>
-
-        <p>This is a starter Flask template. It includes Bootstrap 3, jQuery 2, Flask-SQLAlchemy,
-            WTForms, and various testing utilities out of the box.</p>
-        <p><a href="https://github.com/sloria/cookiecutter-flask"
-              class="btn btn-default btn-large">Learn more &raquo;</a></p>
-    </div><!-- /.jumbotron -->
-
     <div class="body-content">
         <div class="row">
+          {% if current_user and current_user.is_authenticated %}
+          <div class="jumbotron">
```

Comment:
This big teal jumbotron thing that you get on start page, when authenticated, what should we do with it?

![screen shot 2017-10-16 at 2 14 09 pm](https://user-images.githubusercontent.com/786326/31611495-96fd1ed0-b27c-11e7-9f6c-b8bdf2a896e4.png)